server/auth/ldap/index.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import * as React from 'react';
+import Router from 'koa-router';
+import { User, Team } from '../../models';
+import auth from '../../middlewares/authentication';
+import md5 from 'blueimp-md5';
+import getUser from './ldapGetUser';
+import LdapSignin from '../../pages/LdapSignin';
+import renderpage from '../../utils/renderpage';
+
+
+const router = new Router();
+
+router.get('ldap', auth({ required: false }), async ctx => {
+  return renderpage(
+    ctx,
+    <LdapSignin
+      notice={ctx.request.query.notice}
+      error={ctx.request.query.error}
+    />
+  );
+});
+
+router.get('ldap.callback', auth({ required: false }), async ctx => {
```

Comment:
Was unable to turn this into a POST request. Wasn't able to parse the body of a POST using koa-bodyparser if someone with more experience could help out here it would be appreciated.